From kuninori.morimoto.gx@gmail.com Tue Jun 12 21:51:16 2012
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Tue, 12 Jun 2012 21:51:14 -0700 (PDT)
Subject: [PATCH 1/2] LTSI: bugfix: armadillo800eva: SH_ETH needs NET_ETHERNET on defconfig
To: Greg KH <gregkh@linuxfoundation.org>, ltsi-dev@lists.linuxfoundation.org
Cc: Simon <horms@verge.net.au>, Kuninori Morimoto <kuninori.morimoto.gx@gmail.com>, (KMC)tuji <kunihiko@kmckk.co.jp>
Message-ID: <87zk877r29.wl%kuninori.morimoto.gx@renesas.com>


On LTSI, sh_eth driver was backported from latest kernel.
But the ethernet driver dependency was modified between
latest kernel and 3.0.x kernel.
Then, LTSI kernel lost NET_ETHERNET which was required
from SH_ETH on defconfig.
This patch fix it up for LTSI.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
---
 arch/arm/configs/armadillo800eva_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/armadillo800eva_defconfig b/arch/arm/configs/armadillo800eva_defconfig
index 0d20749..f29893a 100644
--- a/arch/arm/configs/armadillo800eva_defconfig
+++ b/arch/arm/configs/armadillo800eva_defconfig
@@ -68,6 +68,7 @@ CONFIG_NETDEVICES=y
 # CONFIG_NET_VENDOR_MARVELL is not set
 # CONFIG_NET_VENDOR_MICREL is not set
 # CONFIG_NET_VENDOR_NATSEMI is not set
+CONFIG_NET_ETHERNET=y
 CONFIG_SH_ETH=y
 # CONFIG_NET_VENDOR_SEEQ is not set
 # CONFIG_NET_VENDOR_STMICRO is not set
-- 
1.7.5.4

